presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryMetadata.java

```diff
@@ -47,6 +50,7 @@ public QueryMetadata(
             Optional<String> queryType,
             Optional<String> preparedQuery,
             String queryState,
+            Optional<CatalogSchemaTableName> targetTable,
```

Comment:
What's "target" table? That's supposed to be what "output" captures (in QueryOutputMetadata)